test/e2e/docker_containers.go

```diff
@@ -17,34 +17,25 @@ limitations under the License.
 package e2e
 
 import (
-	"time"
-
 	"k8s.io/kubernetes/pkg/api"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 	"k8s.io/kubernetes/pkg/util"
 
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("Docker Containers", func() {
+	framework := Framework{BaseName: "docker-containers"}
```

Comment:
I think we don't want to change the namespace, so:

s/docker-containers/containers/